examples/simple-lambda.hs

```diff
@@ -40,3 +45,53 @@ code = "\
 \  callback(null, \"some success message\"); \
 \ } \
 \ "
+
+
+role :: Resource
+role = 
+  resource "IAMRole" $
+  IAMRoleProperties $
+  iamRole rolePolicyDocumentObject
+  & iamrPolicies ?~ [ executePolicy ]
+  & iamrRoleName ?~ "MyLambdaBasicExecutionRole"
+  & iamrPath ?~ "/"
+
+  where
+    executePolicy = 
+      iamPolicies
+      (HM.fromList
```

Comment:
heh, that QQ business strikes me as a step back to writing CF template in json 😄 

`object` fn yields a value of type `Value`, i actually need a value of type `Object` here (`HashMap` alias)
Just thought there was a helper similar to `object` to create `Object`s. But it's fine